### PR TITLE
Doxygen: Enhanced Pack documentation about device memory specification.

### PR DESCRIPTION
**Pull request by JonatanAntoni**
Clarified usage of deprecated id and replacing name attributes.
Added missing init and uninit attributes.

### CODE REVIEW

File: CMSIS/DoxyGen/Pack/src/devices_schema.txt [hunk 1]
```diff
@@ -1635,10 +1641,30 @@ This element can be defined on various levels. Inner memory elements supersede o
...
+    <td>uninit (new since version 1.6.3)</td>
+    <td>
+      <p>If \token{true}, the memory region shall be kept uninitialized (i.e. keep the memory state as is).</p>
+      <p>Default value is \token{false}.</p>
+    </td>
+    <td>xs:boolean</td>
+    <td>optional</td>
+  </tr>
+  <tr>
+    <td>alias(new since Version 1.4.0)</td>
```
Comment (KeilChris) [inline CMSIS/DoxyGen/Pack/src/devices_schema.txt:1662]: Needs a space between alias and (

File: CMSIS/DoxyGen/Pack/src/devices_schema.txt [hunk 2]
```diff
@@ -1587,21 +1587,25 @@ This element can be defined on various levels. Inner memory elements supersede o
...
     <td>optional</td>
   </tr>
   <tr>
-    <td>name (new in Version 1.4.0)</td>
-    <td>unique name of the memory (new in Version 1.4.0) to be used in conjunction with <em>access</em>
+    <td>name (new since version 1.4.0)</td>
+    <td>
+      <p>Unique name of the memory to be used in conjunction with <em>access</em>.</p>
+      <p>If a memory with the same name is already defined in a parent scope, the parent one is extended/overwritten.</p>
+      <p>Backward compatibility: If no name attribute is given but id is still present, the given id is used as the name.</p>
```
Comment (KeilChris) [inline CMSIS/DoxyGen/Pack/src/devices_schema.txt:1601]: Please put id (twice) between <em></em> to make it clear that we are talking about the attribute "id".

File: CMSIS/DoxyGen/Pack/src/devices_schema.txt [hunk 3]
```diff
@@ -1635,10 +1641,30 @@ This element can be defined on various levels. Inner memory elements supersede o
...
+      <p>If \token{true}, the memory region shall be kept uninitialized (i.e. keep the memory state as is).</p>
+      <p>Default value is \token{false}.</p>
+    </td>
+    <td>xs:boolean</td>
+    <td>optional</td>
+  </tr>
+  <tr>
+    <td>alias(new since Version 1.4.0)</td>
+    <td>
+      Reference to another memory (by name) which shares the same physical memory. Some physical
```
Comment (KeilChris) [inline CMSIS/DoxyGen/Pack/src/devices_schema.txt:1664]: Are you referring to the attribute "name" in "by name"? Then put in <em></em>?

**Comment by flit** [comment]
Thanks 😄

**Pull request merged**